docs/deployment/nsgate.md

```diff
@@ -1,2 +1,29 @@
 # Nsgate - Gcloud Cluster Front Loadbalancer
-TODO
+There are different approach to connect to a kubernetes cluster.
+- [kubernetes service resource][service-official-doc]
+- [kubernetes ingress resource][ingress-official-doc]
+
+The solution we use is the **kubernetes service** resource. It's a load balancers level 4.  
+The problem you can encounter is that a service comes with an external IP. That external IP can be generated when you create the service or reserved in advance. But how to predict the numbers of environments you need. You also have to manage the new dns records to access your environments. These records then have to propagate ...
```

Comment:
I don't understand this part: 'But how to predict the numbers of environments you need.' - is it a question?